cdk/lib/rule-manager/rule-manager.ts

```diff
@@ -0,0 +1,178 @@
+          ],
+          resources: ["*"]
+        }),
+        new PolicyStatement({
+          effect: Effect.ALLOW,
+          actions: [
+            "ssm:GetParametersByPath"
+          ],
+          resources: [`arn:aws:ssm:${this.region}:${this.account}:parameter/${this.stage}/${this.stack}/typerighter-rule-manager`]
+        })
+      ]
+    })
+
+    const pandaAuthPolicy = new GuPolicy(this, "PandaAuthPolicy", {
+      policyName: "PandaAuthPolicy",
+      statements: [
+        new PolicyStatement({
+          effect: Effect.ALLOW,
+          actions: [
+            "s3:GetObject"
+          ],
+          resources: ["arn:aws:s3:::pan-domain-auth-settings/*"]
+        })
+      ]
+    })
+
+    const ruleManagerRole = new InstanceRole(this, {
+      artifactBucket: "composer-dist",
+      additionalPolicies: [simpleCofnigPolicy, pandaAuthPolicy]
+    });
+
+    const targetGroup = new GuApplicationTargetGroup(this, "PublicTargetGroup", {
+      vpc: vpc,
+      port: 9000,
+      protocol: ApplicationProtocol.HTTP,
+      targetType: TargetType.INSTANCE,
+      healthCheck: {
+        port: "9000",
+        protocol: Protocol.HTTP,
+        path: "/healthcheck",
+        interval: Duration.minutes(1),
+        timeout: Duration.seconds(3),
+        healthyThresholdCount: 2,
+        unhealthyThresholdCount: 5,
+      },
+      deregistrationDelay: Duration.seconds(30),
+    });
+
+    const ingressRules = {
+      "Global": "0.0.0.0/0"
+    }
+
+    const loadBalancerSecurityGroup = new GuSecurityGroup(this, "LoadBalancerSecurityGroup", {
+      description: "Security group to allow internet access to the LB",
+      vpc,
+      allowAllOutbound: false,
+      ingresses: transformToCidrIngress(Object.entries(ingressRules))
+    });
+
+    const privateSubnets = GuVpc.subnets(this, parameters.PrivateSubnets.valueAsList);
+    const publicSubnets = GuVpc.subnets(this, parameters.PublicSubnets.valueAsList);
+
+    const loadBalancer = new GuApplicationLoadBalancer(this, "PublicLoadBalancer", {
+      vpc,
+      internetFacing: true,
+      vpcSubnets: { subnets: publicSubnets },
+      securityGroup: loadBalancerSecurityGroup,
+    });
+
+    new GuApplicationListener(this, "PublicListener", {
+      loadBalancer,
+      certificates: [{ certificateArn: parameters.TLSCert.valueAsString }],
+      defaultAction: ListenerAction.forward([targetGroup]),
+      open: false,
+    });
+
+    const appSecurityGroup = new GuSecurityGroup(this, "ApplicationSecurityGroup", {
+      description: "HTTP",
+      vpc,
+      allowAllOutbound: true,
+    });
+
+    const userData = `#!/bin/bash -ev
+mkdir /etc/gu
+
+cat > /etc/gu/typerighter-rule-manager.conf <<-'EOF'
+    include "application"
+EOF
+
+aws --quiet --region ${this.region} s3 cp s3://composer-dist/${this.stack}/${this.stage}/typerighter-rule-manager/typerighter-rule-manager.deb /tmp/package.deb
+dpkg -i /tmp/package.deb`;
+
+    new GuAutoScalingGroup(this, "AutoscalingGroup", {
```

Comment:
A lot of boilerplate! `@guardian/cdk` should do this for free, hopefully we'll address this soon!